### PR TITLE
Fix evolution error for Hitmonlee, Hitmonchan, Dratini

### PR DESCRIPTION
**Pull request by jebabin**
Fix evolution error for Hitmonlee, Hitmonchan, Dratini

### CODE REVIEW

**Comment by amal** [comment]
Fixed in 3455

File: data/pokemon.json [hunk 1]
```diff
@@ -5724,6 +5712,12 @@
     ],
     "Weight": "3.3 kg",
     "Height": "1.8 m",
+    "Next evolution(s)": [
```
Comment (Anakin5) [inline data/pokemon.json]: missing second evolution

**Comment by jebabin** [comment]
@amal sorry I did not see it. But even in #3455 there is still an error for Hitmonlee who is not evolving to Hitmonchan.
My pull miss the 2nd evolution of Dratini so I'll close it and re submit a corrected version.

File: data/pokemon.json (hunk 1 above)
Comment (jebabin) [inline data/pokemon.json]: Fixed

**Comment by TheSavior** [comment]
Did we lose hitmonchan and hitmonlee in this PR?

**Comment by amal** [comment]
@jebabin yes, thank you! Fixed hitmonchan and hitmonlee info in #3455

**Comment by TheSavior** [comment]
Seems like this is covered by #3455. Thanks for the PR though @jebabin

**Pull request closed without merging**